samples/node-token-validation-samples/response-sample/index.js

```diff
@@ -0,0 +1,205 @@
+            if (req.query.loginHint) authCodeUrlParameters.loginHint = req.query.loginHint;
+
+            // Check for the domainHint parameter
+            if (req.query.domainHint) authCodeUrlParameters.domainHint = req.query.domainHint;
+        }
+
+        req.session.nonce = authCodeUrlParameters.nonce //switch to a more persistent storage method.
+        
+        /**
+         * MSAL Usage
+         * The code below demonstrates the correct usage pattern of the ClientApplication.getAuthCodeUrl API.
+         * 
+         * Authorization Code Grant: First Leg
+         * 
+         * In this code block, the application uses MSAL to obtain an authorization code request URL. Once the URL is
+         * returned by MSAL, the express application is redirected to said request URL, concluding the first leg of the
+         * Authorization Code Grant flow.
+         */
+        clientApplication.getAuthCodeUrl(authCodeUrlParameters).then((authCodeUrl) => {
+            res.redirect(authCodeUrl);
+        }).catch((error) => console.log(JSON.stringify(error)));
+    });
+
+    app.get("/redirect", (req, res) => {
+        const tokenRequest = { ...requestConfig.tokenRequest, code: req.query.code };
+        const authCodeResponse = { nonce: req.session.nonce, code: req.query.code }
+
+        /**
+         * MSAL Usage
+         * The code below demonstrates the correct usage pattern of the ClientApplication.acquireTokenByCode API.
+         * 
+         * Authorization Code Grant: Second Leg
+         * 
+         * In this code block, the application uses MSAL to obtain an Access Token from the configured authentication service.
+         * The cached nonce is passed in authCodeResponse object and shall later be validated by MSAL once the Access Token and ID 
+         * token are returned.
+         * The response contains an `accessToken` property. Said property contains a string representing an encoded Json Web Token
+         * which can be added to the `Authorization` header in a protected resource request to demonstrate authorization.
+         */
+
+        clientApplication.acquireTokenByCode(tokenRequest, authCodeResponse).then((response) => {
+            console.log("Successfully acquired token using Authorization Code.");
+            
+            /**
+             * Id Token options to be passed into the TokenValidator.validateTokenFromResponse API.
+             * 
+             * A separate object for access token options can also be passed in to the API to validate access tokens on the response.
+             */
+            const idTokenOptions = {
+                validIssuers: config.validationParams.idTokenOptions.validIssuers,
+                validAudiences: config.validationParams.idTokenOptions.validAudiences,
+                nonce: response.idTokenClaims.nonce
+            };
+
+            /**
+             * Uncomment accessTokenOptions and add to line 138 if wanting to validate access token with custom scopes
```

Comment:
Curious if this is part of the actual code

Reply:
Could you clarify your question? These commented out lines are to demonstrate to someone using the sample how they could add `accessTokenOptions` to validate an access token returned on the response.